fission/main.go

```diff
@@ -124,6 +218,35 @@ func main() {
 				return nil
 			},
 		},
+		{
+			Name:  "get-env-variables",
```

Comment:
Yes. Current implementation is not easy to use. But we should find a way to cache the fission url or we need to fetch the url every time cli runs.